README.md

```diff
@@ -154,7 +149,25 @@ Kubeless also supports [ingress](https://kubernetes.io/docs/concepts/services-ne
 
 ### PubSub function
 
-A function can be as simple as:
+NOTE: Set of Kafka-Zookeeper are required to deploy pubsub function at this moment. Please consider to use our provided manifest to deploy kafka and zookeeper statefulsets, as Kubeless hasn't been configured to integrate with existing kafka/zookeeper system. Kafka statefulset uses a PVC (persistent volume claim). Depending on the configuration of your cluster you may need to provision a PV (Persistent Volume) that matches the PVC or configure dynamic storage provisioning. Otherwise Kafka pod will fail to get scheduled. Please refer to [PV](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) documentation on how to provision storage for PVC.
```

Comment:
I would make a small introduction first about "what is" a pubsub function.

`Please consider to use our provided manifest to deploy kafka and zookeeper statefulsets, as Kubeless hasn't been configured to integrate with existing kafka/zookeeper system` -> `We provide manifests to make Kubeless work out of the box with Kafka and Zookeeper`. And then we can show the commands to deploy the manifest.

Reply:
Added a sentence to explain kubeless pubsub function and removed the duplicate note